src/component/Calculator.js

```diff
@@ -1,39 +1,108 @@
 /* eslint-disable  */
 import React from 'react';
+import PropTypes from "prop-types";
 
-
-const Calculator = ({ obj, handleClick }) => {
-  const { total, operation, next } = obj;
+class Calculator extends React.Component {
+   render() {
```

Comment:
- [SUGGESTION] 

In this component, we get the following linter error:

`Component should be written as a pure function eslint(react/prefer-stateless-function)`

This can be solved by adding a class constructor to the Calculator class component

```JavaScript
class Calculator extends React.Component {
  constructor(props) {
      super(props);
      this.state = {};
  }

   render() {
     // render logic goes here
   }
}
```